docs/en/CLI.md

```diff
@@ -90,7 +91,7 @@ abp new Acme.BookStore
   * **`console`**: [Console template](Startup-Templates/Console.md).
 * `--output-folder` or `-o`: Specifies the output folder. Default value is the current directory.
 * `--version` or `-v`: Specifies the ABP & template version. It can be a [release tag](https://github.com/abpframework/abp/releases) or a [branch name](https://github.com/abpframework/abp/branches). Uses the latest release if not specified. Most of the times, you will want to use the latest version.
-* `--preview`: Use latest pre-release version (Only if `--version ` is not specified and there is at least one pre-release after latest stable version).
+* `--preview`: Use latest preview version (Only if `--version` is not specified and there is at least one pre-release after latest stable version).
```

Comment:
Remove "(Only if `--version` is not specified and there is at least one pre-release after latest stable version)"